Fixed isECCertsCMExist & isECSecretExists function

mvenukadasula opened the pull request:
- Fixed the logic in helpers.tpl > isECCertsCMExist & isECSecretExists functions to check for ec-certs and ec-secret objects exists in the cluster before installing the helm chart.
- Removed IF condition in deployment.yaml to check ec-certs and ec-secret before creating the pod.  Argo-cd might be using helm template and not evaluating this condition whehn synced.  Created an issue in GitHub for future tracking.

mvenukadasula committed:
Fixed isECCertsCMExist & isECSecretExists function

Fixed the logic in sECCertsCMExist & isECSecretExists functions to check for ec-certs and ec-secret objects exists in the cluster before installing the helm chart.

mvenukadasula commented:
@ayasuda-ge please hold on merging this PR.  Still testing the change in argo-cd.

naddym reviewed (approved):
Thanks @mvenukadasula, LGTM +1 to EC-Release:v1 👍

mvenukadasula committed:
Removed IF condition not evaluated by argo-cd

Removed IF condition to check ec-certs and ec-secret before creating the pod.  Argo-cd might be using helm template and not evaluating this condition whehn synced.  Created an issue in GitHub for future tracking.

mvenukadasula commented:
These changes have been tested from my repo: https://github.build.ge.com/digital-connect-devops/ec-eng-argo-cd-apps/blob/master/ec-services/ec-service-3dc7ad45/Chart.yaml

Here is the test result: https://argocd-preprod-aws.digitalconnect.apps.ge.com/applications/ec-service-9e768876
